src/topology/separation.lean

```diff
@@ -120,10 +120,12 @@ class t1_space (α : Type u) [topological_space α] : Prop :=
 lemma is_closed_singleton [t1_space α] {x : α} : is_closed ({x} : set α) :=
 t1_space.t1 x
 
+lemma is_open_ne [t1_space α] {x : α} : is_open {y | y ≠ x} :=
+by simpa using is_open_neg (t1_space.t1 x)
```

Comment:
Was there a PR that changed the naming convention, so `is_open_neg` should now be `is_open_compl`?

Reply:
`is_open_neg` uses `set_of`. We also have `is_open_compl_iff`.

Reply:
After e8a68de `is_open_neg` is never used in `mathlib`.